plugin/core/views.py

```diff
@@ -33,6 +33,8 @@
 import sublime_plugin
 import tempfile
 
+MarkdownLangMap = Dict[str, Tuple[Sequence[str], Sequence[str]]]
```

Comment:
```suggestion
MarkdownLangMap = Dict[str, Tuple[Tuple[str], Tuple[str]]]
```

@rchl  is right about tuples. They are crucial to make the langmap to be accepted by mdpopups.